net-misc/ntpsec/ntpsec-9999.ebuild

```diff
@@ -164,12 +153,18 @@ src_install() {
 
 	# move doc files to /usr/share/doc/"${P}"
 	use doc && mv -v "${ED}"/usr/share/doc/"${PN}" "${ED}"/usr/share/doc/"${P}"/html
-}
-
-python_install() {
+	local impl
+	for impl in "${_PYTHON_SUPPORTED_IMPLS[@]}"; do
+		local instpath=${d#${D%/}}
+		instpath=/${instpath##/}
+		has "${impl}" "${PYTHON_COMPAT[@]}" && \
+		use "python_targets_${impl}" && \
+		mkdir -pv "${D}/usr/lib/${impl/_/.}/site-packages/ntp/" && \
```

Comment:
Please use the [helpers](https://devmanual.gentoo.org/eclass-reference/python-utils-r1.eclass/index.html) from python-utils-r1.eclass.

Reply:
I think it would be great if I could replace a bunch of my cargo cult glue with something like the following:

```
python_foreach_impl \
    ${EPYTHON} -m gpep517 \
    install-wheel "$(wheel_name}" \
    --optimize all --destdir "${D}"
```

I do not think I can because that would cd into another environment and futz several environment variables.